pypit/arparse.py

```diff
@@ -4004,7 +4046,7 @@ def key_int(v):
     return v
 
 
-def key_keyword(v):
+def key_keyword(v, force_format=True):
```

Comment:
add doc on force_format

Reply:
done - I had forgotten what this was for  :-S